components/docs-chef-io/content/automate/ha_troubleshooting.md

```diff
@@ -180,4 +180,23 @@ terraform taint module.bootstrap_automate.null_resource.automate_post[1]
 terraform taint module.bootstrap_automate.null_resource.automate_post[2]
 ```
 
+### Issue: Deployment or Upgrade Fails as UnknownError
 
+While trying to deploy or upgrade Chef Automate HA, if the command did not exit gracefully
+
+The possible error looks like as shown below:
```

Comment:
While trying to deploy or upgrade Chef Automate HA, if the command did not exit gracefully, the possible error looks like as shown below:

(Merge these two lines)

Reply:
Done